### PR TITLE
Update pr template

### PR DESCRIPTION
**Pull request by asnaith**
Added a new check to prompt us about billing changes being applied to both Files and Storage.

### CODE REVIEW

**Comment by asnaith** [comment]
I'm not sure how I upset the code analysis with such a small change, I followed the same indentation 😅. I'll look again tomorrow.

**Comment by asnaith** [comment]
Codacy seems to be failing with two (incorrect?) warnings on the markdown file. 

I have experimented with their suggested way to ignore files and added a codacy.yml config at the root to exclude the `.github` directory but it didn't work.

Does anyone have any suggestions to suppress these warnings?

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -10,5 +10,9 @@ Submission checklist:
 - [ ] Change looks good in the mobile web ui
 
 #### Theme
-- [ ] Components / elements inspected in light mode 
-- [ ] Components / elements inspected in dark mode 
+- [ ] Components / elements inspected in light mode
+- [ ] Components / elements inspected in dark mode
+
+#### Billing
+- [ ] Change applied to both files ui and storage ui
```
Comment (Tbaut) [inline .github/pull_request_template.md]:
```suggestion
- [ ] Change applied to both files-ui and storage-ui
```

**Review by Tbaut — COMMENTED** [review]
I guess it's drunk, let's re-run it

**Review by Tbaut — APPROVED** [review]
Looks good now :)

edit: erm no

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -10,5 +10,9 @@ Submission checklist:
 - [ ] Change looks good in the mobile web ui
 
 #### Theme
-- [ ] Components / elements inspected in light mode 
-- [ ] Components / elements inspected in dark mode 
+- [ ] Components / elements inspected in light mode
+- [ ] Components / elements inspected in dark mode
+
+#### Billing
+- [ ] Change applied to both files-ui and storage-ui
```
Comment (Tbaut) [inline .github/pull_request_template.md:17]:
```suggestion
-[ ] Change applied to both files-ui and storage-ui
```

**Comment by Tbaut** [comment]
I deactivated those warnings on codacy directly, they are 100% not helpful and wrong IMO

**Review by FSM1 — APPROVED** [review]

**Comment by asnaith** [comment]
@Tbaut Thanks for taking care of this

**Pull request merged**